2018/Chuvagina/NewTournament/Tournament/BinarySaver.cs

```diff
@@ -18,11 +18,38 @@ public static void SaveListToBinnary<T>(String fileName, List<T> serializableObj
 
         public static List<T> LoadListFromBinnary<T>(String fileName)
```

Comment:
I do not think that we need to know from the outside, about the names of the files